libs/auth-api-lib/src/lib/delegations/delegations.service.ts

```diff
@@ -538,27 +628,66 @@ export class DelegationsService {
         provider: DelegationProvider.PersonalRepresentativeRegistry,
       })
 
-      const rp = await this.prService.getByPersonalRepresentative(
+      const personalRepresentatives = await this.prService.getByPersonalRepresentative(
```

Comment:
We should filter inactive personal representatives so we're not constantly rechecking them. This should be inside `getByPersonalRepresentative`, maybe a flag to skip or include inactive personal representatives.

Reply:
good point